modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -153,6 +157,14 @@ func (a *ActiveHealthChecks) Provision(ctx caddy.Context, h *Handler) error {
 	a.httpClient = &http.Client{
 		Timeout:   timeout,
 		Transport: h.Transport,
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			if !a.HealthFollowRedirects {
+				return errors.New(
+					"Redirects are disabled in health check, set health_follow_redirects flag in config to avoid this error")
+			} else {
+				return nil
+			}
```

Comment:
The `else` is unnecessary

```suggestion
			}
			return nil
```